.eslintrc.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
```

Comment:
You should update this so it uses the same 2 spaces of indentation the rest of the project does